package/MDAnalysis/analysis/pca.py

```diff
@@ -294,19 +294,19 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
 
         Parameters
         ----------
-        atomgroup: MDAnalysis atomgroup/ Universe
+        atomgroup : MDAnalysis atomgroup/ Universe
             The atomgroup or universe containing atoms to be PCA transformed.
-        n_components: int, optional
-            The number of components to be projected onto, Default none: maps
+        n_components : int, optional
+            The number of components to be projected onto, Default None: maps
             onto all components.
-        start: int, optional
+        start : int, optional
             The frame to start on for the PCA transform. Default: None becomes
             0, the first frame index.
-        stop: int, optional
+        stop : int, optional
             Frame index to stop PCA transform. Default: None becomes n_frames.
             Iteration stops *before* this frame number, which means that the
             trajectory would be read until the end.
-        step: int, optional
+        step : int, optional
```

Comment:
rewrite, if set to ``None`` (the default) then every frame is analyzed (i.e., same as ``step=1``) -- makes things clearer

Reply:
Why didn't we set the default to 1?

Reply:
Because using `None` as step = 1 is the behavior for slicing and range so we keep the user interface "pythonic".

Reply:
@PicoCentauri do you want to improve this doc string, too, or are we squash-merging as is?